### ISSUE TITLE
dry run

### ISSUE THREAD

**Issue by nmccready** [sam-cross/delete-google-music-duplicates#1]
Is there a dry option to make things are not obliterated ahead of time?

**Comment by transiient** [comment]
It tells you what's gonna be deleted when you run it - you have to confirm before it deletes anything.

I can add a safety mode with output to a text file! I'll start working on that.

**Comment by nmccready** [comment]
Thanks I should have read into the code in more depth. But cool that your into making it safer.

**Comment by transiient** [comment]
Yeah no worries, that's actually a feature which could be helpful for some.
